std/builtin/logic.neva

```diff
@@ -0,0 +1,26 @@
+#extern(match)
+pub component Match<T>(data T, [case] T) ([case] T, else T)
+
+#extern(eq)
+pub component Eq<T>(actual T, expected T) (res bool)
+
+#extern(if)
+pub component If(data bool) (then any, else any)
+
+#extern(not)
+pub component Not(data bool) (res bool)
+
+#extern(and)
+pub component And(A bool, B bool) (res bool)
+
+#extern(or)
+pub component Or(A bool, B bool) (res bool)
+
+#extern(int int_is_greater, float float_is_greater)
+pub component IsGreater<T int | float>(actual T, compared T) (res bool)
```

Comment:
1. Let's call it just `Gt`
2. Should also be able to compare `strings` (you don't have to implement runtime function, just add to overloading)

Reply:
1. Sure

2. That doesnt match though? Youre not expecting it to be greater youre just checking

Reply:
1. Should be consistent with `Eq`. `Gt/Lt/Eq` are pretty common naming you can find in many places. You can also add comment that will explain that Gt stands for "Greater Than"
2. This is not how they are compared in almost any other language including Go. Try to compare two characters by their length
3. It could mean "expected to greater than this". Don't make user think, it will be easier with consistent naming. This also about "Greater vs Gt". User will see `Eq` and will expect it to be `Gt/Lt`, not Greater/Less